releases/Process/Major Release Process.md

```diff
@@ -59,91 +56,108 @@ The release process is managed by sig-release. To request an update to this docu
 Each of the below phases is represented on the Github Issues project board for the release. 
 
 ## Phase (1 of 4): Planning
-*Intent: Identify all stakeholders and their requirements for the release. Confirm stakeholder responsibilities.* 
-Begins 3 months prior to estimated release.
-Entry Criteria: sig-release has has discussed the need for a release, per current schedule. 
+*Intent: Identify all stakeholders and their requirements for the release. Confirm stakeholder responsibilities.
+Begins: 3 months prior to estimated release.
+Entry Criteria: Release SIG has has indicated the need for a release and has a general idea of the release date. See [key details.](#key-details) for the current release cadence.*
 
 ### 1.1 General Steps
- 1. **sig-release** coordinates with the Technical Steering Committee, the O3DE Foundation, and the sigs to determine a release date. This may be done by the SIG chair, co-chair, or the release manager is one has been chosen.
+ 1. **Release SIG** coordinates with the Technical Steering Committee, the O3DE Foundation, and the sigs to determine a release date. This may be done by the SIG chair, co-chair, or the release manager is one has been chosen.
```

Comment:
Does the marketing committee is considered part of the O3DE Foundation? @tonybalandiuk

Reply:
Good catch. I updated it to explicitly mention the "Open 3D Foundation Marketing Committee"